.eslintrc.yml

```diff
@@ -21,5 +21,7 @@
     semi-spacing: 2
     strict: 2
     no-var: 2
+    prefer-template: 2
+    template-curly-spacing: 2
```

Comment:
can you also add `no-template-curly-in-string: 2`?

Reply:
Sure! It makes no difference, though 🙂